ext/vendor/mpack/mpack.c

```diff
@@ -1,26 +1,26 @@
 /**
```

Comment:
We leave sources in the `ext/vendor` dir as-is so we don't need to `clang-format` any of the sources in this dir.

Reply:
Uhm, I just did `make clang_format_fix` (i.e. `make clang_find_files_to_lint` returned these files) ... shall I then exclude them?

Reply:
Did you run that locally on your Mac? That target is supposed to [ignore these paths](https://github.com/DataDog/dd-trace-php/blob/281bc42/Makefile#L147-L152) but I think there are some incompatibilities with `find` on Mac. (At least I vaguely remember running into a similar issue where a `find` command would return different results from the containers.)

Reply:
Yes, I did. Perhaps I should fix that then to not have it happen again accidentally.

Reply:
Fixed the find expression to return the same files (different order, but that doesn't matter :-D) and re-applied.

Reply:
Thanks for fixing that!